### PR TITLE
docs: add badges for build status, license and auto-wiki

### PR DESCRIPTION
**Pull request by turadg**
_incidental_

## Description

The auto-wiki from Mutable.ai is quite useful. Provide a link: https://wiki.mutable.ai/Agoric/agoric-sdk

While adding that badge I noticed we lacked a couple other important ones so this adds those too.

### Security Considerations
none

### Scaling Considerations
none

### Documentation Considerations
nothing more

### Testing Considerations
Reviewers, take a look at the rendered README

### Upgrade Considerations
none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 # Agoric Platform SDK
 
+[![build status](https://github.com/Agoric/SES-shim/workflows/CI/badge.svg?branch=master)](https://github.com/Agoric/SES-shim/actions?query=workflow%3ACI)
```
Comment (ivanlei) [inline README.md]: _please fix_:

Looks like this is badging about the SES-shim repo not agoric-sdk

**Review by ivanlei — DISMISSED** [review]
Looks good in preview. 1 copy/paste error it seems.

File: README.md (hunk 1 above)
Comment (turadg) [reply README.md]: good catch!

File: README.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 # Agoric Platform SDK
 
+![unit tests status](https://github.com/Agoric/agoric-sdk/actions/workflows/test-all-packages.yml/badge.svg)
+![integration tests status](https://github.com/Agoric/agoric-sdk/actions/workflows/integration.yml/badge.svg)
```
Comment (kriskowal) [inline README.md:6]: Currently red, fwiw.

**Review by kriskowal — APPROVED** [review]

**Pull request merged**